website/source/docs/platform/k8s/helm/run.html.md

```diff
@@ -196,6 +196,47 @@ $ kubectl exec -ti <name of pod> -- vault operator unseal
 After a few moments the Vault cluster should elect a new active primary.  The Vault
 cluster is now upgraded!
 
+#### Protecting Sensitive Vault Configurations
```

Comment:
I'm wondering if this should be an h3 instead of an h4 heading? Otherwise this is appearing under the `Upgrading Vault on Kubernetes` section. Though that would also mean bumping the other heading below up one as well.
```suggestion
### Protecting Sensitive Vault Configurations
```